app/src/main/res/layout/activity_home.xml

```diff
@@ -0,0 +1,33 @@
+<android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_home_drawer_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fitsSystemWindows="true"
+    android:orientation="vertical"
+    tools:context="com.android.template.ui.main.MainActivity">
+
+    <FrameLayout
+        android:id="@+id/activity_home_fragment_container"
```

Comment:
Mislim da sam vać komentirao ovo, ali kao bolje je da svi activity-ji imaju isti ID za fragment container, jer onda možeš zakucati jedan ID u Router klasi.

Al ak si odlučio da tebi to ne paše, sve 5

Reply:
ahaaaaaaaaa ja sam mislio da globalno u svim layoutima mora biti različit ID

Reply:
izmijenit ću to